data/speakers.ts

```diff
@@ -1236,7 +1237,7 @@ const speakers: Speaker[] = [
     },
     talks: [],
```

Comment:
Hi @manekinekko, thanks for your PR! Could you please add your talks, future, and past. If you have too many of them, just last 5 for instance :) See Nicolas Hoizey as an example, please. Thank you very much again!

Reply:
Hi, I'm merging this for now, but could you please update your talks in the new PR? Thank you.